pkg/netceptor/tlsconfig.go

```diff
@@ -106,12 +139,13 @@ func (cfg tlsServerCfg) Prepare() error {
 
 // tlsClientConfig stores the configuration options for a TLS client.
 type tlsClientConfig struct {
-	Name               string   `required:"true" description:"Name of this TLS client configuration"`
-	Cert               string   `required:"false" description:"Client certificate filename"`
-	Key                string   `required:"false" description:"Client private key filename"`
-	RootCAs            string   `required:"false" description:"Root CA bundle to use instead of system trust"`
-	InsecureSkipVerify bool     `required:"false" description:"Accept any server cert" default:"false"`
-	PinnedServerCert   []string `required:"false" description:"Pinned fingerprint of required server certificate"`
+	Name                   string   `required:"true" description:"Name of this TLS client configuration"`
+	Cert                   string   `required:"false" description:"Client certificate filename"`
+	Key                    string   `required:"false" description:"Client private key filename"`
+	RootCAs                string   `required:"false" description:"Root CA bundle to use instead of system trust"`
+	InsecureSkipVerify     bool     `required:"false" description:"Accept any server cert" default:"false"`
+	PinnedServerCert       []string `required:"false" description:"Pinned fingerprint of required server certificate"`
+	SkipReceptorNamesCheck bool     `required:"false" description:"if enabled, skip verifying ReceptorNames OIDs in certificate at startup"`
```

Comment:
golang will default this to false, but for consistency and clarify I think we can add `default:"false"` , similar to InsecureSkipVerify

To make the description a little shorter, we could omit "if enabled," and just start with "Skip.."